### DISCUSSION TITLE
Spacy Tutorial not working in Google Colab

### DISCUSSION THREAD

**Discussion by mgt-strategies** [explosion/spaCy#11361] [Help: Installation] [answered]
spaCy_tuTorial.ipynb
bombs out  - mostly from version incompatibilities with WordNet I think.

**Comment by ljvmiranda921** [comment, marked as answer]
Hi @mgt-strategies , we might need more information to help you solve the problem. 

First off, would you mind sharing the exact code to reproduce the problem (is this based on a Jupyter notebook somewhere or some blog post tutorial?). It would also be great if you paste the exact error here, so we can see what actually happened.

Also note that we don't think we authored the notebook you mentioned. Contacting the original author to help you resolve the problem is also a good idea!
